### DISCUSSION TITLE
private PlayerTarget

### DISCUSSION THREAD

**Discussion by ElTuna** [EssentialsX/Essentials#5114] [Q&A] [answered]
I've been trying to access the `name` field on this class to get the player name an user is teleporting to, but for some reason it's set to private:
https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/main/java/com/earth2me/essentials/PlayerTarget.java#L8
Any reason as to why this is not public?

**Comment by JRoy** [comment, marked as answer]
Will be resolved with https://github.com/EssentialsX/Essentials/pull/5131
